### ISSUE TITLE
Today new error - r4590 - ERROR ConnServer GetFile exception: AttributeError: __enter__ in FileRequest.py line 227

### ISSUE THREAD

**Issue by slrslr** [ZeroNetX/ZeroNet#174]
### Step 1: Please describe your environment

  * ZeroNet version: 0.8.0 (rev4590)
  * Operating system: Linux
  * Web browser: Firefox

### Step 2: Describe the problem:

@canewsin You have made a typo during today revision release?
I am having tens of thousands errors of this kind:
```
ERROR Site:1ADQAH..g6aA File Access error: FileNotFoundError: [Errno 2] No such file or directory: './data/1ADQAHsqsie5PBeQhQgjcKmUu3qdPFg6aA/data/users/userhere/src/imagehere.jpg' in SiteStorage.py line 259
ERROR ConnServer GetFile exception: AttributeError: __enter__ in FileRequest.py line 227

ERROR Site:1DdPHe..Vi9D File Access error: FileNotFoundError: [Errno 2] No such file or directory: './data/1DdPHedr5Tz55EtQWxqvsbEXPdc4uCVi9D/data/users/userhere/filehere.png' in SiteStorage.py line 259
[2022-05-26 21:51:42,809] ERROR ConnServer GetFile exception: AttributeError: __enter__ in FileRequest.py line 227
```

or files was somehow deleted from my data folder?

**Comment by canewsin** [comment]
This is because of missing file errors. Till now those are treated as unhandled exceptions. Could you confirm that, this is the case of missing files!

**Comment by slrslr** [comment]
I have checked a few files that was reported that way and are indeed missing.
Today i have supplied the ZN data folder to zeronet-conservancy by @caryoscelus
Maybe it invalidated content.db or something that ZN was rechecking sites and reporting errors (not enabled verbose log to tell what happened during startup). I have long time not seen such high count of errors.
If files was not deleted by zeronet and never existed, then ZN should not flood the log by such errors i think.

**Comment by canewsin** [comment]
**Fixed in #174**
